tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -1318,6 +1327,23 @@ public function userHidesTheSharedResourceUsingTheGraphApi(string $user): void {
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When user :user hides the federated share :share using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $share
+	 *
+	 * @return void
+	 * @throws Exception
+	 * @throws GuzzleException
+	 */
+	public function userHidesTheFederatedShareUsingTheGraphApi(string $user, string $share): void {
+		$remoteItemId = $this->spacesContext->getSharesRemoteItemId($user, $share);
+		$remoteItemId = \rawurlencode($remoteItemId);
```

Comment:
do we need to encoding because other place i see using remoteItemID without encoding ?